index.html

```diff
@@ -62,5 +62,198 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+   <section class="grid-container">
+            <section class="project-section">
+                <div>
+                    <h2 class="headings">Projects</h2>
+                </div>
+        
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop1.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
```

Comment:
Please kindly reduce the font weight of the title to match the provided using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design
![image](https://user-images.githubusercontent.com/78251927/236285341-10d8da6f-8046-434e-80b5-30b4b1058b8b.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/236285112-b572bec9-ad35-49e0-8a21-560a98782f3b.png)